report_templates/registered_users.erb

```diff
@@ -0,0 +1,26 @@
+<%#
+name: Registered users
+description: Report of registered users
+snippet: false
+template_inputs:
+- name: Users filter
+  required: false
+  input_type: user
+  description: Limit the report only on users found by this search query. Keep empty
+    for report on all available users.
+  advanced: false
+  value_type: search
+  resource_type: User
+model: ReportTemplate
+-%>
+<%- load_users(search: input('Users filter'), includes: :auth_source).each_record do |user| -%>
+  <%-  report_row({
```

Comment:
the indentation here is incorrect, it should start on col 0 and the indentation should be done inside of the ERB, see other reports for an example